lib/spack/llnl/util/tty/__init__.py

```diff
@@ -24,6 +26,7 @@
 _warn_enabled = True
 _error_enabled = True
 indent = "  "
+_osascript = os.path.exists('/usr/bin/osascript')
```

Comment:
Make this lazy (i.e., wrap it in a function call) so that it is not called unconditionally on module import.  Doing lots of things in module scope slows down Spack startup.